ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -562,7 +562,7 @@ private static VerifiableProperties buildFrontendVProps(File trustStoreFile)
     // to test that multipart requests over a certain size fail
     properties.put("netty.multipart.post.max.size.bytes", Long.toString(MAX_MULTIPART_POST_SIZE_BYTES));
     CommonTestUtils.populateRequiredRouterProps(properties);
-    TestSSLUtils.addSSLProperties(properties, "", SSLFactory.Mode.SERVER, trustStoreFile, "frontend");
+    TestSSLUtils.addSSLProperties(properties, "", SSLFactory.Mode.SERVER, trustStoreFile, "frontend", null);
```

Comment:
Minor: Looks like we calling addSSLProperties with null argument a lot of times. Can we just overload the function and call one from another?

Reply:
done!